extensions/wikia/AdEngine/js/video/player/porvata/porvata.js

```diff
@@ -64,6 +64,15 @@ define('ext.wikia.adEngine.video.player.porvata', [
 				log(['porvata video player created', video], log.levels.debug, logGroup);
 				tracker.register(video, params);
 
+				function shouldResume(isVisible) {
+					// Don't resume when video was paused manually
+					return isVisible && autoPaused
+						// Do not resume automatically when video has controls
+						&& !videoSettings.hasUiControls()
+						// Do not resume when video floating is active
+						&& !isFloatingEnabled(params);
+				}
```

Comment:
per coding conventions (https://github.com/Wikia/guidelines/blob/master/PHP/CodingConventions.md#conditional-logic) please move `&&`s to the EOLs